README.md

```diff
@@ -1,4 +1,14 @@
-# Marlin 3D Printer Firmware

```

Comment:
You probably don't want to modify the main Marlin `README.md` file. You can put a `README.md` into the folder with your configurations if there's any special info that needs inclusion. (See `example_configurations/K8200/readme.md` as an example of one that was included with a config.)